### ISSUE TITLE
Graph not loading on gnome-terminal

### ISSUE THREAD

**Issue by Lacrymology** [lusingander/serie#54]
I don't get the graph at all on gnome-terminal. I tried both `-p` values,  and using `--preload`.

![image](https://github.com/user-attachments/assets/23a08e52-4445-432d-adc9-a380d7f4d1ad)

**Comment by alerque** [comment]
It has been a long time since I used it much, but I don't recall Gnome Terminal ever having any image protocol support. Does it now? If so which image protocol(s) are expected to work?

**Comment by tompropst** [comment]
I don't believe the gnome terminal has image protocol support. I see the same behavior in `tmux` for the same reason. If it is practical to support an option where the graph is rendered like `git log --graph` but with the navigation features of `serie`, that would be awesome.

**Comment by ericbn** [comment]
Wouldn't it be nice to have an option to show the graph in the original ASCII format used by git? I'm on iTerm and the experience is so slow that I'd trade not having the fancier graph in return of a fully responsive interface without thinking twice.

I love the idea of having a CLI tool just for nicely browsing the git log and quickly switching between the log and a diff view (or any custom view as you're now enabling with custom commands).
